.github/workflows/updatejson.yml

```diff
@@ -0,0 +1,27 @@
+name: Housekeeping
+on: 
+  push:
+  schedule:
+  #  Kick off job at midnight on the 1st and the 15th of every month
+  - cron: "0 0 1,15 * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps: 
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Comment:
We should use python3. python2 has reached EOL.

Will this install the dependencies automatically?

Reply:
So the setup-python@v2 doesn't mean its installing python2. It means its installing python using the 2nd version of the already created **setup-python** action. You can see it here https://github.com/actions/setup-python

I've just double checked and the Python version that is installed is 3.9.0. I'm fairly sure it'll install the dependencies necessary to run the script on the virtual machine just fine

Reply:
Ahh. That's my bad. Thanks for the clarification.